pkg/plugins/storage/storage/storage.go

```diff
@@ -18,22 +18,41 @@ import (
 	"context"
 	"fmt"
 	"io/ioutil"
+	"time"
 
 	ifaces_gcloud_storage "github.com/nitric-dev/membrane/pkg/ifaces/gcloud_storage"
+	"github.com/nitric-dev/membrane/pkg/utils"
 
 	"cloud.google.com/go/storage"
 	"github.com/nitric-dev/membrane/pkg/plugins/errors"
 	"github.com/nitric-dev/membrane/pkg/plugins/errors/codes"
 	plugin "github.com/nitric-dev/membrane/pkg/plugins/storage"
 	"golang.org/x/oauth2/google"
+	"golang.org/x/oauth2/jwt"
 	"google.golang.org/api/iterator"
 	"google.golang.org/api/option"
 )
 
+type StorageUtils interface {
+	JWTConfigFromJSON(jsonKey []byte) (*jwt.Config, error)
+	SignedURL(bucket string, key string, opts *storage.SignedURLOptions) (string, error)
+}
+
+type StorageStorageUtils struct{}
```

Comment:
this looks a bit odd (I really don't know the code yet, so please excuse me if I am off base), any way you could have:
```go
type storageUtils struct{} // lower case

// also good to have this as it makes it more obvious what struct implements what interface
var _ StorageUtils = &storageUtils{}
```